include/nbl/asset/filters/CNormalMapToDerivativeFilter.h

```diff
@@ -0,0 +1,328 @@
+// Copyright (C) 2018-2020 - DevSH Graphics Programming Sp. z O.O.
+// This file is part of the "Nabla Engine".
+// For conditions of distribution and use, see copyright notice in nabla.h
+
+#ifndef __NBL_ASSET_C_NORMAL_MAP_TO_DERIVATIVE_FILTER_H_INCLUDED__
+#define __NBL_ASSET_C_NORMAL_MAP_TO_DERIVATIVE_FILTER_H_INCLUDED__
+
+#include "nbl/core/core.h"
+
+#include <type_traits>
+#include <functional>
+ 
+#include "nbl/asset/filters/CMatchedSizeInOutImageFilterCommon.h"
+#include "nbl/asset/filters/CSwizzleAndConvertImageFilter.h"
+#include "CConvertFormatImageFilter.h"
+
+namespace nbl
+{
+namespace asset
+{
+
+template<typename Swizzle>
```

Comment:
you should probably take a template parameter for the dither as well

Reply:
corrected in following commit

Reply:
where is that commit?